ecosystem/typescript/sdk/src/transaction_builder/builder.ts

```diff
@@ -33,6 +33,9 @@ import * as Gen from "../generated/index";
 
 export { TypeTagParser } from "./builder_utils.js";
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Can we just name it according to the convention to satisfy this lint?

Reply:
done